Sources/AppCoinsSDK/UI/Purchase/ViewModels/TransactionViewModel.swift

```diff
@@ -147,7 +148,7 @@ internal class TransactionViewModel : ObservableObject {
         }
     }
     
-    private func getPaymentMethods(value: String, currency: Coin, completion: @escaping ([PaymentMethod]) -> Void) {
+    private func getPaymentMethods(value: String, currency: String, completion: @escaping ([PaymentMethod]) -> Void) {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done